webserver/cWebem.cpp

```diff
@@ -2195,13 +2067,22 @@ namespace http {
 
 			// Initialize session
 			WebEmSession session;
+			session.id = "";
+			session.username = "";
+			session.rights = -1;
+			session.reply_status = reply::ok;
+			session.isnew = false;
```

Comment:
No need to initialize variables that are already initialized in the constructor of the object. (also below)